marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -165,6 +170,19 @@ class BrowsePage {
    return cy.get('#selected-facets [data-cy="clear-'+ lowerBound +'"]');
   }
 
+  clickPopoverSearch(facetName: string){
+      return cy.get(`[data-testid=${facetName}-search-input]`).click();
```

Comment:
we can remove return here and in the locator below. No need to return when action is performed in the locator itself. We only "return" an element locator.